packages/transformers/js/src/lib.rs

```diff
@@ -258,13 +261,16 @@ fn transform(ctx: CallContext) -> Result<JsUnknown> {
             let module = {
               let mut passes = chain!(
                 // Inline process.env and process.browser
-                EnvReplacer {
-                  replace_env: config.replace_env,
-                  env: config.env,
-                  is_browser: config.is_browser,
-                  decls: &decls,
-                  used_env: &mut result.used_env
-                },
+                Optional::new(
+                  EnvReplacer {
+                    replace_env: config.replace_env,
+                    env: config.env,
+                    is_browser: config.is_browser,
+                    decls: &decls,
+                    used_env: &mut result.used_env
+                  },
+                  config.source_type != SourceType::Script
```

Comment:
Should process.env be supported in scripts?

Reply:
I guess it would be nice? But we could also add that later if it comes up. The only thing we can't support is the `process` Node global replacement.

Reply:
Mainly I was just trying to make scripts as close to real browser JS as possible, and not enable any CommonJS/Node features.